### PR TITLE
add missing Clone on BmpParsingContext

### PR DESCRIPTION
**Pull request by mxyns**
BmpParsingContext does not derive Clone

### CODE REVIEW

File: crates/bgp-pkt/src/capabilities.rs [hunk 1]
```diff
@@ -338,6 +338,10 @@ impl AddPathCapability {
     pub const fn address_families(&self) -> &Vec<AddPathAddressFamily> {
         &self.address_families
     }
+
+    pub fn address_families_mut(&mut self) -> &mut Vec<AddPathAddressFamily> {
```
Comment (ahassany) [inline crates/bgp-pkt/src/capabilities.rs:342]: what's this for? I tried to keep the packets immutable. Also unrelated to this PR

File: crates/iana/src/address_family.rs [hunk 2]
```diff
@@ -417,6 +417,7 @@ pub enum AddressType {
     Ipv6FlowSpecL3Vpn,
     Ipv6NlriMplsLabels,
     L2VpnBgpEvpn,
+    L2VpnVpls,
```
Comment (ahassany) [inline crates/iana/src/address_family.rs:420]: unrelated to this PR, please put it in another one

File: crates/bmp-pkt/src/lib.rs [hunk 3]
```diff
@@ -1028,6 +1028,27 @@ pub struct PeerKey {
     bgp_id: Ipv4Addr,
 }
 
+impl Hash for PeerKey {
```
Comment (ahassany) [inline crates/bmp-pkt/src/lib.rs:1031]: unrelated to this PR

**Review by ahassany — CHANGES_REQUESTED** [review]
This PR need some cleanup to keep what's related to the BmpParsingContext clone implementation

**Comment by mxyns** [comment]
i'll open different ones

**Pull request closed without merging**